response_operations_ui/views/collection_exercise.py

```diff
@@ -136,7 +136,12 @@ def view_collection_exercise(short_name, period):
     else:
         show_set_live_button = ce_state in "READY_FOR_REVIEW"
 
-    show_sds = True if ce_details["collection_exercise"]["supplementaryDatasetEntity"] else False
+    if ("supplementaryDatasetEntity" in ce_details["collection_exercise"]) and (
```

Comment:
Use get to tidy this up, it will look after the existence and a None return

```
show_sds = True if ce_details["collection_exercise"].get("supplementaryDatasetEntity") else False

Reply:
Collection exercise shouldn't return a None in the sds value, I had just left it there if one snuck through, the test data has been updated to reflect the lack of value or the sds json arriving, and the if has been updated with a get.